docs/src/main/asciidoc/security-jpa.adoc

```diff
@@ -80,12 +80,13 @@ public class User extends PanacheEntity {
 
 The `quarkus-security-jpa` extension initializes only if a single entity is annotated with `@UserDefinition`.
 
-<1> The `@UserDefinition` annotation must be present on a single entity, either a regular Hibernate ORM entity or a Hibernate ORM with Panache entity.
-<2> Indicates the field used for the username.
-<3> Indicates the field used for the password.
+<1> In most databases, `user` is a restricted keyword, so define the table name as something else to avoid conflicts
+<2> The `@UserDefinition` annotation must be present on a single entity, either a regular Hibernate ORM entity or a Hibernate ORM with Panache entity.
+<3> Indicates the field used for the username.
+<4> Indicates the field used for the password.
 By default, `quarkus-security-jpa` uses bcrypt-hashed passwords, or you can configure plain text or custom passwords instead.
```

Comment:
IMHO this note is not needed here, there is a dedicated section, https://quarkus.io/guides/security-jpa#password-storage-and-hashing.
Perhaps, updating the current line to `Indicates the field used for the password. See the <<password-hashing-and-storage>> for more information` can work ? (`password-hashing-and-storage` anchor should be present in that section or refer to the existing one if it is already available)